### PR TITLE
Create add_a_banner

### PR DESCRIPTION
**Pull request by kelea99**
This is a starting off point for a "request a banner" template for DACs in orangelight.

### CODE REVIEW

**Review by christinach — APPROVED** [review]
Thanks @kelea99

File: .github/ISSUE_TEMPLATE/add_a_banner [hunk 1]
```diff
@@ -25,3 +24,6 @@ _Please include hard deadlines and date range for banner implementation_
 
 ### Sudden Priority Justification
 _Required if "asap" is checked._
+
+### Implementation notes
+https://github.com/pulibrary/orangelight#announcement-messages
```
Comment (maxkadel) [inline .github/ISSUE_TEMPLATE/add_a_banner:29]: To make the link more accessible and clear, I'd suggest:

```suggest
[Documentation on how to set new banner messages](https://github.com/pulibrary/orangelight#announcement-messages)
```

**Review by maxkadel — CHANGES_REQUESTED** [review]
Tiny change to make the Implementation note link clearer

**Review by maxkadel — APPROVED** [review]
Thanks @kelea99 🎉 !

**Pull request merged**